test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -58,9 +60,13 @@ string ProtoConverter::createHex(string const& _hexBytes)
 	string tmp{_hexBytes};
 	if (!tmp.empty())
 	{
-		boost::range::remove_erase_if(tmp, [=](char c) -> bool {
-			return !std::isxdigit(c);
-		});
+		tmp.erase(
```

Comment:
I think the previous version was better: No use of iterators and shorter. Why do you want to get rid of it? Maybe there is a ranges alternative?

Reply:
The ranges alternative is sadly more verbose :(

I simply removed this particular change.

Reply:
I wanted to remove boost dependencies because I felt we wanted to eventually use standard library alternatives over boost. I guess that needs to wait.